app/templates/main/browse-series.html

```diff
@@ -124,25 +131,26 @@ <h3 class="govuk-heading-s govuk-heading-s--browse-all-filter">Date consignment
                 {% for record in results %}
                     <tr class="govuk-table__row">
                         <td class="govuk-table__cell">
-                            <div class="browse__table__series-row">
-                                <a href="{{ url_for('main.browse', series_id=record['series_id']) }}">{{ record["series"]
-                                }}</a>
-                            </div>
                             <div>{{ record["last_record_transferred"] }}</div>
                         </td>
                         <td class="govuk-table__cell">
-                            <div class="browse__table__series-row empty-cell">-</div>
                             <div class="right-align">{{ record["records_held"] }}</div>
                         </td>
                         <td class="govuk-table__cell">
-                            <div class="browse__table__series-row right-align empty-cell">-</div>
                             <div class="right-align">
-                                <a href="{{ url_for('main.browse', consignment_id=record['consignment_id']) }}">{{ record["consignment_reference"] }}</a>
+                                <a href="{{ url_for('main.browse_consignment', _id=record['consignment_id']) }}">{{ record["consignment_reference"] }}</a>
                             </div>
                         </td>
                     </tr>
                 {% endfor %}
             </tbody>
         </table>
+        <!-- PAGINATION -->
+        <div class="govuk-grid-column-two-thirds browse-grid--two-thirds">
```

Comment:
so were we previously completely missing pagination for all these views?

Reply:
pagination was on browse.html for common use, it has been moved to individual browse view to pass in _id parameter based on that view

Reply:
Ah yes, I see 👌